libraries/apollo-gradle-plugin-tasks/src/main/kotlin/com/apollographql/apollo/gradle/task/apolloGenerateCodegenSchema.kt

```diff
@@ -11,7 +11,7 @@ import gratatouille.tasks.GTask
 @GTask
 internal fun apolloGenerateCodegenSchema(
     logger: GLogger,
-    arguments: Map<String, GAny?>,
+    arguments: Map<String, GAny>,
```

Comment:
It's not 100% clear whether Gradle `MapProperty` supports nullable values. To remove any ambiguity, I made this non-nullable.